src/components/EntityPage/File/index.js

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { compose } from 'recompose';
+import { injectState } from 'freactal';
+import { withTheme } from 'emotion-theming';
+
+import styled from 'react-emotion';
+import Row from '../../../uikit/Row';
+import Column from '../../../uikit/Column';
```

Comment:
I think absolute path reference would be better for uikit :P `uikit/Row` should work